app/connectors/RegistrationConnector.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import cats.data.EitherT
+import config.FrontendAppConfig
+import models.register.error.ApiError.{InternalServerError, RecordNotFoundError, RequestCouldNotBeProcessedError, UnableToParseError}
+import models.register.error.{ApiError, RegisterWithIDErrorResponse}
+import models.register.request.PayloadRegisterWithID
+import models.register.response.PayloadRegistrationWithIDResponse
+import play.api.http.Status._
+import play.api.libs.json.{JsError, JsSuccess}
+import uk.gov.hmrc.http.HttpReads.is2xx
+import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpResponse}
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class RegistrationConnector @Inject() (val config: FrontendAppConfig, val http: HttpClient) {
+
+  val submissionUrl = s"${config.businessMatchingUrl}/registration/02.00.00/withId"
```

Comment:
@iservport The URL is not the right one, can you please update it to the correct one